pom.xml

```diff
@@ -45,23 +46,51 @@
   </distributionManagement>
 
   <properties>
-    <jenkins.version>1.642.3</jenkins.version>
+    <jenkins.version>2.121.2</jenkins.version>
```

Comment:
According to [stats](http://stats.jenkins.io/pluginversions/ansicolor.html) this is not a big deal, even if there are active trunk releases of this plugin for unrelated changes.

But, if baseline is an obstacle, the POM changes could be reverted. In that case the only `src/main/` change would be to convert some uses of lambdas to inner classes, pretty simple; the `src/test/` code could remain, but would no longer reproduce the problem unless run under `plugin-compat-tester`.